.gitignore

```diff
@@ -1,7 +1,6 @@
 # Common IDEs
 *.iml
 .idea
-.idea/**/*
```

Comment:
Ignoring this file folder is taken care of on line 3; technically it should be .idea/